pkg/observability/tsconfig.json

```diff
@@ -18,10 +18,7 @@
       "../../node_modules/@rancher/shell/types"
     ],    
     "types": [
-      "node",
-      "webpack-env",
       "@types/node",
-      "@types/jest",
```

Comment:
those are missing in node_modules

Reply:
I did the build and dev load and didn't find any problems, so this should be good